website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -54,24 +54,21 @@ Hang tight while we grab the latest from your chart repositories...
 Update Complete. ⎈Happy Helming!⎈
 ```
 
-<Note title="Helm does not automatically update CRDs">
-  You must update all CRDs manually before upgrading VSO.
-  Refer to <a href="#updating-crds">Updating CRDs</a>.
+<Note title="VSO 0.8.0+ automatically updates CRDs">
+  As of VSO 0.8.0, VSO will automatically update its CRDs.
+  The manual upgrade step <a href="#updating-crds">Updating CRDs</a> below is no longer required when
+  upgrading to VSO 0.8.0+.
```

Comment:
This link doesn't seem to work in [the preview](https://vault-q14p04a2f-hashicorp.vercel.app/vault/docs/platform/k8s/vso/installation#upgrading-using-helm), but a regular markdown link should work if there's blank lines before and after the content:

```suggestion
<Note title="VSO 0.8.0+ automatically updates CRDs">

  As of VSO 0.8.0, VSO will automatically update its CRDs.
  The manual upgrade step [Updating CRDs](#updating-crds) below is no longer required when
  upgrading to VSO 0.8.0+.

```

Reply:
Reworked in 5dca6a516a9b43c84e375d410478c9eeaa94ef23